app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,27 @@
+# Prepares data to be shown to the users for their dashboard.
+class DashboardController < ApplicationController
+  respond_to :html, :js
+
+  def index
+    @partner = current_partner
+  end
+
+  # TODO: Remove commented code below
```

Comment:
Can remove all this commented out code too

Reply:
Removed ✔️